### ISSUE TITLE
add exception to attribute error

### ISSUE THREAD

**Issue by 5u2ie** [Mincka/DMArchiver#59]
after saving about 6 threads the script stops with the error: 'str' object has no attribute 'tweet_id'
probably just a matter of adding an exception but I don't have much experience with building binaries

**Comment by Mincka** [comment]
Hi, I would like to know why the tool fails to find a tweet_id. Would you be able to identify a specific type of tweet in this conversation, that could cause the issue?

**Comment by 5u2ie** [comment]
For the thread that's giving the error I also get the following:

Previous conversation not found. Creating a new one with incremental support.
Begin of thread reached
Total processed tweets: 828

Could it be that the thread is too long?

**Comment by Mincka** [comment]
No, it's not related to the size. The tool managed to download more than 120k in a single thread. It's always for this specific conversations and at 828 tweets that the script fails?

**Comment by 5u2ie** [comment]
I ran it a few times, it worked for about 5 threads then it always got stuck at the same thread which I then deleted. Now it gets stuck at the first thread, which used to work with 'Total processed tweets: 0', but now it's giving an error:

Previous conversation not found. Creating a new one with incremental support.
Begin of thread reached
Total processed tweets: 1
writing conversation to *path*
'str' object has no attribute 'tweet_id'
